### DISCUSSION TITLE
Radar Mode

### DISCUSSION THREAD

**Discussion by odvigajlo** [niels747/2D-Weather-Sandbox#82] [Ideas]
Radar mode, basically this mode will allow to see precipitation like in weather radar. This may seem useless but getting idea of precipitation strength is pretty needed, right now it's really hard to tell how strong precipitation would be in real life. And well it's always interesting to learn storms and their structure, i always wanted to see weather radar image in slice. 

I suggest this working like this:
For all particles, we will count amount of other particles in certain radius, and depending on their amount we will know precipitation strength at this point. Now we make circle with same radius as we use for detecting particles and color it with amount of particles in radius, and then raymarch them. Like this we should get pretty good image of radar. Also all freezing particles should have pink/white color (hail on radar).

**Comment by Meso22** [comment]
Maybe a color mode for precipitation? Purple for heavier sized precipitation and green for light precip.

**Reply by Edcaaaa4gy53** [reply]
Need this idea this instant!
